cli/reporter/azure_service_bus_handler.go

```diff
@@ -49,10 +54,17 @@ func (h *azureSbusHandler) WriteReport(ctx context.Context, report *policy.Repor
 	if err != nil {
 		return err
 	}
+
 	msg := &azservicebus.Message{
 		Body: data,
 	}
-	err = sender.SendMessage(ctx, msg, &azservicebus.SendMessageOptions{})
+	if h.format == JSON {
+		typ := "application/json"
```

Comment:
should we set the right content type for protobuf as well?

Reply:
i dont think we support protobuf here? the format enum doesnt define that